### PR TITLE
Add latest httpcomponents

### PR DESCRIPTION
**Pull request by arunans23**
Add the following latest httpcomponents orbit bundles
- httpcore : 4.4.14
- httpcore-nio : 4.4.14
- httpclient : 4.5.13

### CODE REVIEW

File: httpclient/4.5.12.wso2v1/pom.xml [hunk 1]
```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (Nirothipan) [inline httpclient/4.5.12.wso2v1/pom.xml:3]: Please update the year

File: httpcore-nio/4.4.13.wso2v1/pom.xml [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+       <snapshotRepository>
+           <id>wso2.snapshots</id>
+           <name>Apache Snapshot Repository</name>
+           <url>http://maven.wso2.org/nexus/content/repositories/snapshots/</url>
+       </snapshotRepository>
+   </distributionManagement>
+
+   <build>
+        <plugins>
+			<plugin>
```
Comment (Nirothipan) [inline httpcore-nio/4.4.13.wso2v1/pom.xml:70]: formatting

Comment (arunans23) [reply httpcore-nio/4.4.13.wso2v1/pom.xml]: Fixed

File: httpclient/4.5.12.wso2v1/pom.xml (hunk 1 above)
Comment (arunans23) [reply httpclient/4.5.12.wso2v1/pom.xml]: Fixed

**Review by Nirothipan — APPROVED** [review]

**Pull request merged**